pom.xml

```diff
@@ -1,36 +1,132 @@
-	</properties>
-	<dependencies>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-devtools</artifactId>
-			<scope>runtime</scope>
-			<optional>true</optional>
-		</dependency>
-		
-		<!-- Test -->
-		<dependency>
-			<groupId>se.sundsvall.dept44</groupId>
-			<artifactId>dept44-starter-test</artifactId>
-			<scope>test</scope>
-		</dependency>
-	</dependencies>
-</project>
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>se.sundsvall.dept44</groupId>
+        <artifactId>dept44-service-parent</artifactId>
+        <version>3.2</version>
+        <relativePath/>
+    </parent>
+    <!-- lookup parent from repository -->
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>se.sundsvall</groupId>
+    <artifactId>api-service-private-precheck</artifactId>
+    <version>0.5</version>
+    <name>api-service-private-precheck</name>
+
+    <properties>
+        <docker.image.name>ms-private-precheck</docker.image.name>
+        <!-- Pipeline properties -->
+        <generated-sources-java-path>src/main/java</generated-sources-java-path>
+        <generated-sources-path>${project.build.directory}/generated-sources</generated-sources-path>
+        <!-- Generated sources properties -->
+        <jackson-databind-nullable.version>0.2.6</jackson-databind-nullable.version>
+        <openapi-generator-maven-plugin.version>5.3.1</openapi-generator-maven-plugin.version>
+        <published.container.port.production>xxx</published.container.port.production>
+        <published.container.port.test>xxx</published.container.port.test>
+        <spring-security-jwt.version>1.1.1.RELEASE</spring-security-jwt.version>
+        <openapi-generator-maven-plugin-version>7.1.0</openapi-generator-maven-plugin-version>
+    </properties>
+
+    <dependencies>
+        <!--- openapi-generator-maven-plugin -->
+
+        <dependency>
+            <groupId>org.openapitools</groupId>
+            <artifactId>openapi-generator-core</artifactId>
+            <version>${openapi-generator-maven-plugin-version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-devtools</artifactId>
+            <scope>runtime</scope>
+            <optional>true</optional>
+        </dependency>
+        <!-- Auth -->
+        <dependency>
+            <groupId>org.springframework.security</groupId>
+            <artifactId>spring-security-jwt</artifactId>
+            <version>${spring-security-jwt.version}</version>
+        </dependency>
```

Comment:
Is this dependency needed/used or can it be removed?

Reply:
It is now removed